XC/install/install-xc0.ps1

```diff
@@ -1,5 +1,6 @@
 Param(
-    [string] $ConfigurationFile = '.\configuration-xc0.json'
+    [string] $ConfigurationFile = '.\configuration-xc0.json',
+    [switch] $Blank
```

Comment:
Probably need a better switch name? $Blank doesn't really explain what you're trying to do?

Maybe change the logic and use something like -SkipDefaultAssets or -OverwriteExisting (anything that would be more intuitive is fine.